### DISCUSSION TITLE
free(): double free detected in tcache 2

### DISCUSSION THREAD

**Discussion by karthi-keyan** [fastapi/fastapi#7387] [Questions]
uvicorn main:app --reload
INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [5551] using statreload
INFO:     Started server process [5553]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     127.0.0.1:37034 - "GET /User HTTP/1.1" 200 OK
free(): double free detected in tcache 2

**Comment by phy25** [comment]
It's unlikely that someone will have an idea if you don't want to use the issue template.

**Comment by tiangolo** [comment]
Thanks for the help @phy25 ! :coffee: 

@karthi-keyan we can't help you without more information. If you still have problems, please create a new issue following the issue template step by step.

Given that, I'm gonna close this one.
